style.css

```diff
@@ -145,49 +155,211 @@ li {
 }
 
 .icons-1 {
+  width: 100%;
+  height: 100px;
   display: flex;
   flex-direction: column;
-  height: 100px;
   justify-content: center;
   align-items: center;
+  padding: 0 var(--containers-side-padding);
 }
 
-.icons-2 {
+.links {
   display: flex;
   margin: 40px 0;
   justify-content: center;
   list-style: none;
-  gap: 20px;
 }
 
-.icons-2 ul li {
+.links ul li {
   flex: 0 1 40px;
 }
 
-.icons-2 img {
+.links img {
   background: transparent;
-  width: 15px;
-  height: 15px;
+  width: 30px;
+  height: 30px;
 }
 
-.arrow-left {
+.icons-1 .arrow-image {
+  width: 60px;
+  height: 30px;
+}
+
+.work-left {
   display: none;
 }
 
+.icons-1 .left-arrow {
+  margin-left: 5px;
+}
+
+.icons-1 .right-arrow {
+  margin-right: 5px;
+}
+
+.title-1 {
+  grid-row: 1/2;
+  font-size: 40px;
+  line-height: 52px;
+  color: #9490ff !important;
+  text-transform: uppercase;
+  font-style: normal;
+  width: fit-content;
+  justify-self: center;
+}
+
+.container-4 {
+  display: grid;
+  width: 100%;
+  grid-template-columns: 100%;
+  grid-gap: 1rem;
+  background-image: url('images/myself.png');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  min-height: 100vh;
+}
+
+.container-4 > * {
+  position: relative;
+}
+
+.container-4 .title-1 {
+  border-bottom: 1px solid #646369;
+}
+
+.project {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.p-1 {
+  grid-row: 2/3;
+}
+
+.p-2 {
+  grid-row: 3/4;
+}
+
+.p-3 {
+  grid-row: 4/5;
+}
+
+.p-4 {
+  grid-row: 5/6;
+}
+
+.project > * {
+  align-content: center;
+  width: 100%;
+}
+
+.picture {
+  display: flex;
+  align-content: flex-start;
+  width: var(--wrapper-width);
+  height: auto;
+  justify-content: center;
+  margin: 20px 0;
+}
+
+.project img {
+  width: var(--wrapper-width);
+  height: 100%;
+}
+
+.project-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.project-body > * {
+  width: calc(var(--wrapper-width) - 50px);
+}
+
+.project .title-2 {
+  justify-self: center;
+  margin: 15px 0;
+  padding: 0;
+  font-size: 32px;
+  line-height: 44px;
+  color: white;
+  font-weight: 500;
+  text-align: center;
+}
+
+.project p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  color: #dcdce8;
+  text-align: center;
+}
+
+.skills-4 {
+  display: flex;
+  align-items: center;
+  height: 40px;
+  margin: 20px 0;
+  list-style: none;
+}
+
+.project .skills-4 > * {
+  flex-grow: 1;
+  text-align: center;
+  border-right: 1px #555 solid;
+}
+
+.skills-4 li:last-child {
+  border-right: hidden;
+}
+
+.view {
+  width: calc(var(--wrapper-width) - 50px);
+}
+
+.btn-1 {
+  width: 100px;
+  height: 50px;
+  padding: 10px 10px;
+  background: #9390ff;
+  color: black;
+  align-self: flex-start;
+  border-radius: 12px;
+}
```

Comment:
- [ ] Your button is not matched with the Figma design. You can remove the ` border-radius: 12px;` and change background-color to: `#7D73FC` for matching the Figma design.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![b1](https://user-images.githubusercontent.com/70483835/159106431-af3d3479-1918-46e2-bde2-35db310efa3d.png) | ![b2](https://user-images.githubusercontent.com/70483835/159106444-780ecd0a-c6a1-48ab-b5a1-dd769bd00764.png)